### DISCUSSION TITLE
is defaultIfEmpty import correctly documented ?

### DISCUSSION THREAD

**Discussion by Visible1** [ReactiveX/rxjs#7360] [Report issues other than bug]
I think defaultIfEmpty might be part of rxjs/operators instead of rxjs. Please double check for the example shown on page : https://rxjs.dev/api/index/function/defaultIfEmpty .

**Comment by jakovljevic-mladen** [comment]
All operators are exported from both `'rxjs'` and `'rxjs/operators'` as of v7.2.0. Please take a look at [this commit](https://github.com/ReactiveX/rxjs/commit/6e34d88e7ef7972aac30c41f7e8bc13bd591e046) and [this PR](https://github.com/ReactiveX/rxjs/pull/6488).
